src/components/picker/PickerItem.tsx

```diff
@@ -71,7 +71,10 @@ const PickerItem = props => {
   const _renderItem = () => {
     return (
       <View style={styles.container} flex row spread centerV>
-        <Text numberOfLines={1} style={[styles.labelText, isItemDisabled && styles.labelTextDisabled, labelStyle]}>
+        <Text
+          numberOfLines={1}
+          style={[styles.labelText, isItemDisabled ? styles.labelTextDisabled : undefined, labelStyle]}
```

Comment:
style should move to useMemo with dependency on `isItemDisabled` and `labelStyle`

Reply:
Fixed